warehouse/models/mart/gtfs/fct_daily_scheduled_stops.sql

```diff
@@ -0,0 +1,107 @@
+{{ config(materialized='table') }}
+
+WITH fct_daily_scheduled_trips AS (
+
+    SELECT
+
+        service_date,
+        feed_key,
+        service_id,
+        trip_id,
+        route_id,
+        route_type
+
+    FROM {{ ref('fct_daily_scheduled_trips') }}
+),
+
+dim_stop_times AS (
+    SELECT *
+    FROM {{ ref('dim_stop_times') }}
+),
+
+dim_stops AS (
+    SELECT *
+    FROM {{ ref('dim_stops') }}
+),
+
+dim_routes AS (
+    SELECT *
+    FROM {{ ref('dim_routes') }}
+),
+
+stops_by_day AS (
+
+    SELECT
+
+        trips.service_date,
+        trips.feed_key,
+        CAST(trips.route_type AS INT) AS route_type,
+
+        stop_times.stop_id,
+
+        COUNT(*) AS stop_event_count
+
+    FROM dim_stop_times AS stop_times
+    LEFT JOIN fct_daily_scheduled_trips AS trips
+        ON trips.feed_key = stop_times.feed_key
+            AND trips.trip_id = stop_times.trip_id
+    GROUP BY service_date, feed_key, route_type, stop_id
+),
+
+pivot_to_route_type AS (
+
+    SELECT *
+    FROM
+        (SELECT
+
+            service_date,
+            feed_key,
+            route_type,
+            stop_id,
+            stop_event_count
+
+        FROM stops_by_day)
+    PIVOT(
+        SUM(stop_event_count) AS route_type
+        FOR route_type IN
+        (0, 1, 2, 3, 4, 5, 11)
+    )
+),
+
+fct_daily_scheduled_stops AS (
+    SELECT
+
```

Comment:
Let's add a surrogate key for this and shapes mirroring what we have for trips and add the same tests as well